### ISSUE TITLE
Notification channel has no name

### ISSUE THREAD

**Issue by portalbyte** [FossifyOrg/Phone#196]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Phone/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Phone/issues) or [closed](https://github.com/FossifyOrg/Phone/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Phone/discussions) or [closed](https://github.com/FossifyOrg/Phone/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Phone/blob/master/CONTRIBUTING.md).

### Affected app version

1.2.0

### Affected Android/Custom ROM version

Android 14 / LineageOS 21

### Affected device model

Pixel 7

### How did you install the app?

F-Droid / IzzyOnDroid

### Steps to reproduce the bug

1. Install the app
2. Go to Settings > Apps > Fossify Phone > Notifications
3. Read the name of the (only) channel

### Expected behavior

I'd assume that the main notification channel for the app would be named something like "Calls" or "Incoming Calls".

### Actual behavior

It is called "call_notificatiom_channel".

### Screenshots/Screen recordings

<img alt="ignoreImageMinify" src="https://github.com/FossifyOrg/Phone/assets/168933942/38cb8cd4-f970-4eee-b52d-2e6191ba2b26" width=180 />

### Additional information

_Edit by @naveensingh: triggered image minimizer._

**Comment by zanyshh** [comment]
yeah its same for me too shows the same exact thing for me in notification settings
